src/Tribe/Widget/Widget_Interface.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace Tribe\Widget;
+
+use Tribe\Events\Views\V2\View_Interface;
+
+/**
+ * Interface Widget_Interface
+ *
+ * @since   TBD
+ *@package Tribe\Widget
+ *
+ */
+interface Widget_Interface {
+
+	/**
+	 * Constructor for V2 Widgets.
+	 *
+	 * @since TBD
+	 *
+	 * @param string              $id_base         Optional. Base ID for the widget, lowercase and unique. If left empty,
```

Comment:
```suggestion
	 * @param string              $id_base         Optional. Base ID for the widget, lowercase. If left empty,
```
The last part makes this one redundant?